game.py

```diff
@@ -20,8 +20,34 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
-
+    # random_word_generator = RandomWord()
+    # snowman_word = random_word_generator.word(
+    # word_min_length=SNOWMAN_MIN_WORD_LENGTH, 
+    # word_max_length=SNOWMAN_MAX_WORD_LENGTH)
+
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
+        user_input = get_letter_from_user(correct_letter_guess_statuses, wrong_guesses_list)
+        
+        if user_input in correct_letter_guess_statuses:
+            print("You guessed a letter that's in the word!")
+            correct_letter_guess_statuses[user_input] = True
+        else:
+            print(f"The letter {user_input} is not in the word")
+            wrong_guesses_list.append(user_input)
+
+        print_snowman_graphic(len(wrong_guesses_list))
+        print_word_progress_string(snowman_word, correct_letter_guess_statuses)
+        print(f"Wrong guesses: {', '.join(wrong_guesses_list)}")
+
+        if is_word_guessed(snowman_word, correct_letter_guess_statuses):
+            print('Congratulations, you win!')
+            return
```

Comment:
Nice use of the `return` keyword. This allows for us to exit not only the while loop early if this condition is met but also the function as a whole.